.travis.yml

```diff
@@ -1,38 +1,19 @@
-dist: trusty
-sudo: false
-language: ruby
-rvm:
-- 2.2.2
-addons:
-  apt:
-    packages:
-    - python-dev
-    - python-pip
-    - python-lxml
-    - libcairo2
-    - libpango1.0-0
-    - libgdk-pixbuf2.0-0
-    - libffi-dev
-    - shared-mime-info
-    - python-virtualenv
+services:
+  - docker
+
+before_install:
+  - docker pull pandoc/latex:latest
+  - docker run --volume "`pwd`:/data" --entrypoint '/data/docker.sh' pandoc/latex:latest
 script:
-- virtualenv --setuptools ./venv
-- . ./venv/bin/activate
-- mkdir ~/.fonts/
-- cp -a assets/*.otf ~/.fonts/
-- pip install -U --ignore-installed pip
-- pip install -U --ignore-installed setuptools
-- pip install -r requirements.txt
-- make
-- make deploy
+  - make pandoc-deploy
 deploy:
   skip_cleanup: true
   local_dir: output
   region: us-west-2
   provider: s3
   access_key_id: AKIAJWS554QGKBTMSBCA
   secret_access_key:
-    secure: DvvRo4SBpjLMR1C02CzxK6Ah2y8jPA+44MjtbngF33D9V9GEzEhUNw29eGSJApitYQyWMjIDJcbkrFiQ7EDpNz+PbqJCrNBEXzPJ86hliEafOVs68XinvIl0T4b74HBxGFpyvCs/3ghhQST8eAT7eClfwlfsgNYEcEsNZqtAOP96S3UbCG1hKn2wuuAwN4XVVGB1qazx2vztpIAdTwio8L4i/YlEX08cP58m86X7EZY5w/edZbfAm6nbN/aX5hOg3unNisXu0DvurCi6kR5Sxdg5edRZuhfKujMu2hLqj56jBsQ79tVtkrvEXue9tlyum8LYJxogkhlya9TuQb3a02jGv4nS+6E+lSmxtyHiexFWx0xSVie03OOmHZnPmCVMnZuT+HviguRNFSb4WsLJv1wW7XRxug/m3Brz2kb1MOdo3DrRkZDyEXIOIMpunttQvDgS6RTnAlaqRFndWamPTwGQxRQKRwJ3c/ZRjqd2+pZ3eTvERBH7Lbue/rKkC6hhNgRsQJkaJkN57HT2JQgjlfzo/azKuYyiNuoj0aU8l1/Bj6ZjBgzetaJedKl0/maVld/MJkbUjyrOhxy45Jshv5fECJErtrmOG8Ui5ozmwM97z+whDgkox7lc07KW+8LkZShMzdxYLzZ1wElsTtE1d3vgFNmzarcoW9uveI3eryM=
+    secure: DvvRo4SBpjLMR1C02CzxK6Ah2y8jPA+44MjtbngF33D9V9GEzEhUNw29eGSJApitYQyWMjIDJcbkrFiQ7EDpNz+PbqJCrNBEXzPJ86hliEafOVs68XinvIl0T4b74HBxGFpyvCs/3ghhQST8eAT7eClfwlfsgNYEcEsNZqtAO
```

Comment:
what secrets are you deleting here? Or, I suppose, "what sort of secrets" since if you tell me what the secrets are, they aren't really secret anymore ;)

Reply:
I think that got auto-truncated by mistake. I'm also pretty sure that's the key for pushing this stuff to S3...which should probably be migrated to some more secure method. :)

Reply:
I’m not sure I follow? That seems fine (the travis encryption approach to push artifacts)